app/src/main/java/xyz/klinker/blur/launcher3/Launcher.java

```diff
@@ -502,6 +511,15 @@ protected void onCreate(Bundle savedInstanceState) {
         }
     }
 
+  public void showGlobalSearch() {
+        startSearch(null, false, null, true);
+    }
+
+    public void g(){
```

Comment:
I have a Pixel. Always better to use the built in `Intents`, rather than trying to define it explicitly. Looks good now